Userland/Applications/PDFViewer/main.cpp

```diff
@@ -6,25 +6,30 @@
  */
 
 #include "PDFViewerWidget.h"
+#include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibFileSystemAccessClient/Client.h>
 #include <LibGUI/Application.h>
 #include <LibGUI/Icon.h>
 #include <LibGUI/Menubar.h>
-#include <LibGUI/MessageBox.h>
 #include <LibGUI/Window.h>
 #include <LibMain/Main.h>
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    auto app = GUI::Application::construct(arguments);
+    TRY(Core::System::pledge("stdio recvfd sendfd rpath unix"));
+
+    auto app = TRY(GUI::Application::try_create(arguments));
```

Comment:
I'm surprised this does anything :thinking:
at this point any pledge call will pretend to succeed, but should only give you `stdio rpath`...unless my impl of `no_error` is flawed

Reply:
yep, that was an impl error: https://github.com/SerenityOS/serenity/pull/13327